src/Microsoft.Health.Dicom.Core/Messages/CustomTag/DeleteCustomTagResponse.cs

```diff
@@ -0,0 +1,11 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Dicom.Core.Messages.CustomTag
```

Comment:
Why do we have a empty class?

Reply:
Following https://github.com/microsoft/dicom-server/blob/main/src/Microsoft.Health.Dicom.Core/Messages/Delete/DeleteResourcesResponse.cs.